src/controllers/expense.controller.js

```diff
@@ -0,0 +1,171 @@
+'use strict';
+
+const expenseService = require('../services/expense.service.js');
+const userService = require('../services/user.service.js');
+
+const getAll = (req, res) => {
+  const {
+    userId: recivedUserId,
+    categories,
+    from,
+    to,
+  } = req.query;
+  const userId = Number(recivedUserId);
+
+  let expenses = expenseService.getAll();
+
+  if (!expenses.length) {
+    res.send([]);
```

Comment:
If the array is empty you send manually create an empty array, that doesn't make sense